lib/core/src/lib/dialogs/unsaved-changes-dialog/unsaved-changes-dialog.component.html

```diff
@@ -1,29 +1,35 @@
-<h1 mat-dialog-title class="adf-unsaved-changes-dialog-title">
-    {{ 'CORE.DIALOG.UNSAVED_CHANGES.TITLE' | translate }}
+<h1 mat-dialog-title class="adf-unsaved-changes-dialog-header">
+    {{ (data?.headerText ? data.headerText : 'CORE.DIALOG.UNSAVED_CHANGES.TITLE') | translate }}
```

Comment:
Can we move the checks to somewhere in the ts? I'm not 100% sure, but I have a feeling that having checks like this in the HTML might cause them to be reevaluated on every change detection cycle and reduce performance. And they don't seem like data that would change once the dialog gets opened, so no need to reevaluate them everytime